nextflow.config

```diff
@@ -10,10 +10,15 @@ params {
     modules {
         samtools = 'samtools/0.1.19'
         manta = 'manta/0.27.1'
+        fermikit = 'fermikit/r178'
+        vcftools = "vcftools/0.1.14"
+        tabix = "tabix/0.2.6"
     }
     programs {
         svvcf2bed = "/home/rajohvik/Work/wgs-structvar/SVvcf2bed.pl"
     }
     threads = 16
+    genome_size = "3g"
```

Comment:
Feels like  these two should be prefixed with `fermi_`, since they seem to be rather specific to that tool? Otherwise "genome_size" and "readlen" seems wildly ambigious.